README.md

```diff
@@ -1,48 +1,70 @@
-# Welcome to Project Mercury
+# ![logo][]AI Shell (a.k.a Project Mercury)
 
-**Project Mercury** contains our latest CLI tool that provides an interactive shell session to chat
-with language models, creating an *AI Shell*. Users can use _agents_ to interact with different AI models, or other
-assistance providers, in a conversational manner. **Project Mercury** also provides a framework for
-creating AI agents.
+Welcome to the **AI Shell** repository! AI Shell is a CLI tool that brings the power of artificial
+intelligence directly to your command line! Designed to help you get command assistance from various
+AI assistants, AIShell is a versatile tool to help you become more productive in the command line.
```

Comment:
>  AIShell is a versatile tool 

Here you use `AIShell` instead of `AI Shell`. Not sure if this matters but want to bring it to your attention.